src/typescript-service.ts

```diff
@@ -429,6 +429,7 @@ export class TypeScriptService {
 		const symbols = iterate(configs)
 			.map(config => this._collectWorkspaceSymbols(config, query || symbolQuery, limit))
 			.flatten<SymbolInformation>()
+			.filter(symbol => !symbol.location.uri.includes('/node_modules/'))
```

Comment:
> You mean if you pass a symbol query that is defined in a dependency?

No, I mean that `this._collectWorkspaceSymbols` takes `limit` so it probably returns up to N symbols but then they might be filtered out by the added `filter` call so there might be less results